Clinical-Genomics/cg
# Only upload if family is part of cust00{2,3,4}

This PR whitelists customers for upload to loqusdb.

How to test:
0. install on stage (add instructions on how to as needed)

1. run `scripts/add_loqus_upload_column.sql` on cg-stage (user your fav mysql client)
1. on rasta: `us`
1. `cg upload observations cuddlyoryx` -> should say that cust000 is not whitelisted for uploading
1. set cust003 to be able to upload to loqusdb by setting loqus_access to true in clinical-api-stage
1. `cg upload observations simpleoryx`

Expected outcome:
It goes into the upload code now :)
